src/test/java/HmWrk14.java

```diff
@@ -0,0 +1,30 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class HmWrk14 extends BaseTest{
+    @Test
+    public static void registrationNavigation() {
+
+        WebDriver driver = new ChromeDriver();
+        try {
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+            String url = "https://bbb.testpro.io/";
+            driver.get(url);
+
+            WebElement regLink = driver.findElement(By.id("hel"));
+            regLink.click();
+
+            WebElement regbutton = driver.findElement(By.cssSelector("[type='submit']"));
+            Assert.assertTrue(regbutton.isDisplayed());
```

Comment:
Click Registration linkValidate that you are redirected to Registration page

For example (assert if the current url is equal to your stored (String)  registration page url)

```
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.Assert;
import org.testng.annotations.Test;

import java.time.Duration;

public class HmWrk14 {
    @Test
    public static void registrationNavigation()  {
        WebDriver driver = new ChromeDriver();
        try {
            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
            String url = "https://bbb.testpro.io/";
            String registrationURL = "https://bbb.testpro.io/registration.php";
            
            driver.get(url);
            
            WebElement regLink = driver.findElement(By.id("hel"));
            regLink.click();

            Assert.assertEquals(driver.getCurrentUrl(), registrationURL);
        } finally {
            driver.quit();
        }
    }
}
```

![image](https://user-images.githubusercontent.com/104761482/199925261-c5477b82-6730-4c14-878f-0dd6476f98a5.png)

![image](https://user-images.githubusercontent.com/104761482/199925332-553b0581-e9fb-4502-bb5c-52f79599b95e.png)